Fix issue with strongSwan client container starting and conflicting with ports from strongSwan server container

As seen here: https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true

```
Run docker-compose up -d
[7](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:8)
Creating network "security_internet" with the default driver
[8](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:9)
Creating network "security_intranet" with the default driver
[9](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:10)
Pulling strongswan (strongx509/strongswan:5.9.6)...
[10](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:11)
5.9.6: Pulling from strongx509/strongswan
[11](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:12)
Digest: sha256:5d4626765a411a5b46241e69fd33b56e08c01b0eca190698840ff53a022233b4
[12](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:13)
Status: Downloaded newer image for strongx509/strongswan:5.9.6
[13](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:14)
Pulling opi-security-server (golang:alpine)...
[14](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:15)
alpine: Pulling from library/golang
[15](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:16)
Digest: sha256:9c621254fe1ff5a00291eb689b1be2ce4744eee678e33189f2fd7b4fc726cff4
[16](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:17)
Status: Downloaded newer image for golang:alpine
[17](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:18)
Creating security_strongswan_1 ... 
[18](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:19)
Creating security_strongswan_1 ... done
[19](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:20)
Creating security_opi-security-server_1 ... 
[20](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:21)
Creating security_vpn-client_1          ... 
[21](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:22)
Creating security_vpn-client_1          ... error
[22](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:23)

[23](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:24)
ERROR: for security_vpn-client_1  Cannot start service vpn-client: Address already in use
[24](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:25)
Creating security_opi-security-server_1 ... done
[25](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:26)
Creating security_opi-security-client_1 ... 
[26](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:27)
Creating security_opi-security-client_1 ... done
[27](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:28)

[28](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:29)
ERROR: for vpn-client  Cannot start service vpn-client: Address already in use
[29](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:30)
Encountered errors while bringing up the project.
[30](https://github.com/opiproject/opi-poc/runs/7653451746?check_suite_focus=true#step:5:31)
Error: Process completed with exit code 1.
```

--- a/security/docker-compose.yml
+++ b/security/docker-compose.yml
@@ -3,6 +3,8 @@ version: '3.7'
 services:
   strongswan:
     image: strongx509/strongswan:5.9.6
+    depends_on:
+      - vpn-client
     cap_add:
       - NET_ADMIN
       - SYS_ADMIN
@@ -22,8 +24,6 @@ services:
 
   vpn-client:
     image: strongx509/strongswan:5.9.6
-    depends_on:
-      - strongswan
     cap_add:
       - NET_ADMIN
       - SYS_ADMIN

